tests/plugins/test_terminator.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+Test Terminator
+---------------
+"""
+
+from jinja2 import Template
+
+import folium
+from folium import plugins
+
+def test_terminator():
+    m = folium.Map([45., 3.], zoom_start=1)
```

Comment:
I would be okay, but I would keep it for another PR (maybe `v0.3`).

At the cost of doing this, I'd like to have an object like `PluginTester` so that we do something like :

    class TestTerminator(PluginTester):
        def test_main(self):
            ...

but `PluginTester` would also have a method we could use to generate all the test examples for the doc. I think it's too bad to implement nice maps in the test and have to re-implement it in notebooks for docs and examples.